packaging/deb/freerdp-nightly/rules

```diff
@@ -36,7 +36,7 @@ override_dh_strip:
 
 override_dh_install:
 	mkdir -p debian/tmp/opt/freerdp-nightly/lib/cmake/
-	rm -rf debian/tmp/opt/freerdp-nightly/lib/freerdp/*.a
+	rm -rf debian/tmp/opt/freerdp-nightly/lib/freerdp2/*.a
 
```

Comment:
@hardening that's only the .a files that get created when static channels are enabled and the cmake version is to "old". All winPR libs are installed where they belong.